### PR TITLE
feature: added security section for event type read/write scopes

### PR DESCRIPTION
**Pull request by adyach**

### CODE REVIEW

File: docs/api-spec-generated/security.md [hunk 1]
```diff
@@ -23,5 +23,41 @@ scopes in the future such as limiting access to particular event types and strea
...
 |nakadi.event_stream.read|Grants access for consuming Event streams.|
 
-
-
+<a name="event type"></a>
+### Event Type
+#### Write and read scopes
+
+Nakadi has possibility to check OAuth2 Scopes (scopes) of publisher and consumer to give access
+write and read events only for trusted users.
```
Comment (rcillo) [inline docs/api-spec-generated/security.md:31]: What do you think of "to grant write and read access only for trusted users"?

File: docs/api-spec-generated/security.md [hunk 2]
```diff
@@ -23,5 +23,41 @@ scopes in the future such as limiting access to particular event types and strea
...
+    "type": "json_schema",
+    "schema": "{ \"properties\": { \"order_number\": { \"type\": \"string\" } } }"
+  }
+
+  "write_scopes" : ["oauth2.write.scope.one", "oauth2.write.scope.two"],
+  "read_scopes" : ["oauth2.read.scope.one", "oauth2.read.scope.two"]
+}'
+```
+
+Once publisher tries to write event to `order_received` event type he is examined for scopes
```
Comment (rcillo) [inline docs/api-spec-generated/security.md]: What about "...tries to write _an_ event to `order_receive` _,_ its oath token is examined for scopes..."?

**Comment by rcillo** [comment]
:shipit:

File: docs/api-spec-generated/security.md [hunk 3]
```diff
@@ -23,5 +23,41 @@ scopes in the future such as limiting access to particular event types and strea
...
+Once publisher tries to write an event to `order_received`, its OAuth2 Token is examined for scopes
+which are defined in `write_scopes`. For successful publishing he should have at least one of
+the defined scopes.
+
+The event type owner has to create scopes for the application which is owner of this event type.
+Publishers, in their turn, has to request scopes for their applications, which are defined by
+the event type owner, to be able to publish events.
+
+Application scope validation does not occur if no write or read scopes are specified
+for the event type.
```
Comment (v-stepanov) [inline docs/api-spec-generated/security.md:61]: Probably it makes sense to describe that it is the default behavior

File: docs/api-spec-generated/security.md [hunk 4]
```diff
@@ -23,5 +23,41 @@ scopes in the future such as limiting access to particular event types and strea
...
+  "read_scopes" : ["oauth2.read.scope.one", "oauth2.read.scope.two"]
+}'
+```
+
+Once publisher tries to write an event to `order_received`, its OAuth2 Token is examined for scopes
+which are defined in `write_scopes`. For successful publishing he should have at least one of
+the defined scopes.
+
+The event type owner has to create scopes for the application which is owner of this event type.
+Publishers, in their turn, has to request scopes for their applications, which are defined by
```
Comment (v-stepanov) [inline docs/api-spec-generated/security.md:57]: ha**ve**

File: docs/api-spec-generated/security.md [hunk 5]
```diff
@@ -23,5 +23,41 @@ scopes in the future such as limiting access to particular event types and strea
...
 |nakadi.event_stream.read|Grants access for consuming Event streams.|
 
-
-
+<a name="event type"></a>
+### Event Type
+#### Write and read scopes
+
+Nakadi has possibility to check OAuth2 Scopes (scopes) of publisher and consumer to grant access
+write and read events only for trusted users.
```
Comment (v-stepanov) [inline docs/api-spec-generated/security.md:31]: maybe replace with "trusted services" or "trusted applications" ? Because those are not the real users.

Comment (adyach) [reply docs/api-spec-generated/security.md:31]: @v-stepanov fixed it here https://github.com/zalando/nakadi-manual/pull/39/files

**Pull request merged**